gammapy/utils/fitting/fit.py

```diff
@@ -207,59 +172,130 @@ def covar(self, backend="minuit"):
         result : `CovarResult`
```

Comment:
Add section `Parameters` to the `Fit.covar` docstring?

Reply:
done